content/en/lotus/manage/chain-management.md

```diff
@@ -235,3 +235,155 @@ It is possible to _prune_ the current chain data used by Lotus to reduce the nod
+  HotStoreFullGCFrequency = 20
+
+  # EnableColdStoreAutoPrune turns on compaction of the cold store i.e. pruning
+  # where hotstore compaction occurs every finality epochs pruning happens every 3 finalities
+  # Default is false
+  #
+  # type: bool
+  # env var: LOTUS_CHAINSTORE_SPLITSTORE_ENABLECOLDSTOREAUTOPRUNE
+  EnableColdStoreAutoPrune = false
+
+  # ColdStoreFullGCFrequency specifies how often to performa a full (moving) GC on the coldstore.
+  # Only applies if auto prune is enabled.  A value of 0 disables while a value of 1 will do
+  # full GC in every prune.
+  # Default is 7 (about once every a week)
+  #
+  # type: uint64
+  # env var: LOTUS_CHAINSTORE_SPLITSTORE_COLDSTOREFULLGCFREQUENCY
+  ColdStoreFullGCFrequency = 7
+
+  # ColdStoreRetention specifies the retention policy for data reachable from the chain, in
+  # finalities beyond the compaction boundary, default is 0, -1 retains everything
+  #
+  # type: int64
+  # env var: LOTUS_CHAINSTORE_SPLITSTORE_COLDSTORERETENTION
+  ColdStoreRetention = 0
+  ```
+
+### Operation
+
+When the splitstore is first enabled, the existing blockstore becomes
+the coldstore and a fresh hotstore is initialized.
+
+The hotstore is warmed up on first startup so as to load all chain
+headers and state roots in the current head.  This allows us to
+immediately gain the performance benefits of a smallerblockstore which
+can be substantial for full archival nodes.
+
+All new writes are directed to the hotstore, while reads first hit the
+hotstore, with fallback to the coldstore.
+
+Once 5 finalities have ellapsed, and every finality henceforth, the
+blockstore _compacts_.  Compaction is the process of moving all
+unreachable objects within the last 4 finalities from the hotstore to
+the coldstore. If the system is configured with a discard coldstore,
+these objects are discarded. Note that chain headers, all the way to
+genesis, are considered reachable. Stateroots and messages are
+considered reachable only within the last 4 finalities, unless there
+is a live reference to them.
+
+## Compaction
+
+Compaction works transactionally with the following algorithm:
+- We prepare a transaction, whereby all i/o referenced objects through the API are tracked.
+- We walk the chain and mark reachable objects, keeping 4 finalities of state roots and messages and all headers all the way to genesis.
+- Once the chain walk is complete, we begin full transaction protection with concurrent marking; we walk and mark all references created during the chain walk. On the same time, all I/O through the API concurrently marks objects as live references.
+- We collect cold objects by iterating through the hotstore and checking the mark set; if an object is not marked, then it is candidate for purge.
+- When running with a coldstore, we next copy all cold objects to the coldstore.
+- At this point we are ready to begin purging
+- We then end the transaction and compact/gc the hotstore.
+
+## Garbage Collection
```

Comment:
Consider calling this "Cold Store Garbage Collection".  Maybe to most users it's obvious that GC refers to cold store but to me its ambiguous.